apps/files/src/components/FileLinkSidebar.vue

```diff
@@ -0,0 +1,263 @@
+                <oc-button :aria-label="$_editButtonLabel" @click="$_editPublicLink(link)" icon="edit"/>
+                <oc-button :aria-label="$_deleteButtonLabel" @click="$_removePublicLink(link)" icon="delete"/>
+              </div>
+            </oc-grid>
+          </li>
+        </transition-group>
+        <p class="uk-text-meta" v-if="$_links.length === 0" v-translate>
+          Links can be shared with external collaborators.
+        </p>
+      </template>
+    </div>
+    <div v-else-if="visiblePanel === PANEL_EDIT">
+      <transition enter-active-class="uk-animation-slide-right uk-animation-fast"
+                  leave-active-class="uk-animation-slide-right uk-animation-reverse uk-animation-fast"
+                  name="custom-classes-transition">
+        <div class="uk-position-cover oc-default-background">
+          <edit-public-link :params="params" @close="$_showList()"/>
+        </div>
+      </transition>
+    </div>
+  </div>
+</template>
+<script>
+import { mapGetters, mapActions } from 'vuex'
+import moment from 'moment'
+import mixins from '../mixins'
+
+const EditPublicLink = _ => import('./PublicLinksSidebar/EditPublicLink.vue')
+
+const PANEL_SHOW = 'showLinks'
+const PANEL_EDIT = 'editPublicLink'
+
+export default {
+  mixins: [mixins],
+  components: {
+    EditPublicLink
+  },
+  title: ($gettext) => {
+    return $gettext('Links')
+  },
+  data () {
+    return {
+      visiblePanel: 'showLinks',
+      linkCopied: false,
+
+      // panel types
+      PANEL_SHOW: PANEL_SHOW,
+      PANEL_EDIT: PANEL_EDIT,
+
+      // group for easy payload
+      params: {
+        id: null,
+        name: '',
+        permissions: 1,
+        hasPassword: false,
+        expireDate: null
+      }
+    }
+  },
+  mounted () {
+    if (this.highlightedFile && this.$_links.length === 0) {
+      this.loadLinks({
+        client: this.$client,
+        path: this.highlightedFile.path,
+        $gettext: this.$gettext
+      })
+    }
+  },
+  watch: {
+    highlightedFile (n, o) {
+      if (n === o) {
+        return
+      }
+
+      this.visiblePanel = PANEL_SHOW
+      this.loadLinks({
+        client: this.$client,
+        path: this.highlightedFile.path,
+        $gettext: this.$gettext
+      })
+    }
+  },
+  computed: {
+    ...mapGetters('Files', ['highlightedFile', 'links', 'linksLoading', 'linksError']),
+    ...mapGetters(['getToken', 'capabilities']),
+
+    $_links () {
+      return this.links.filter(link => {
+        return this.compareIds(link.itemSource, this.highlightedFile.id)
+      })
+    },
+
+    $_expirationDate () {
+      const expireDate = this.capabilities.files_sharing.public.expire_date
+
+      return {
+        enabled: !!expireDate.enabled,
+        days: (expireDate.days) ? expireDate.days : false,
+        enforced: expireDate.enforced === '1'
+      }
+    },
+    $_tooltipTextLink () {
+      return `title: ${this.$gettext('Click to open the link')}; pos: bottom`
```

Comment:
no double quoting required for the text ?

Reply:
If you would surround the gettext call with double quotes, they would appear in the tooltip text. I.e. no double quotes required here.